src/testdrive/ci/Dockerfile

```diff
@@ -24,6 +24,8 @@ WORKDIR /workdir
 
 RUN mkdir -p /share/tmp && chmod 777 /share/tmp
 RUN mkdir -p /mzdata && chmod 777 /mzdata
+RUN mkdir -p /scratch && chmod 777 /scratch
```

Comment:
It seems a little weird to me that testdrive is getting access to the scratch directory! Is this strictly necessary, or just convenient based on how mzcompose works?

Reply:
don't think it's necessary! artifact of my flailing to get the tests working, I'll remove and see how it goes